documentation/wiki/Building-Testing-and-Debugging-on-.Net-Core-MSBuild.md

```diff
@@ -20,11 +20,11 @@ Install the latest .NET Core SDK from http://dot.net/core. That will ensure all
 
 ## Build
 
-`./build.sh -skipTests`
+`./build.sh`
 
 ## Tests
 
-`./build.sh`
+`./build.sh -test`
```

Comment:
Can you confirm that for .sh this is `-test` instead of `--test`?

Reply:
Updated with --test.